docs/index.md

```diff
@@ -1,3 +1,41 @@
+
+It was concluded that it would be best to initiate something outside a single project; to create a new API spec with a wider coalition. That would allow users to switch servers (avoiding major loads on a single server or project), provide an opportunity to more easily go beyond the existing gPodder.net API specs and enable developers to address some technical issues with the API specs. Initial discussions on GitHub lead to a meeting on October 2022 with folks from [AntennaPod](https://github.com/AntennaPod/AntennaPod/), [Funkwhale](https://dev.funkwhale.audio/), [Kasts](https://invent.kde.org/multimedia/kasts), [Podfriend](https://github.com/MartinMouritzen/Podfriend) and the [gPodder app for Nextcloud](https://github.com/thrillfall/nextcloud-gpodder).[^initialAPIdiscussions] A few months later, the first of the recurring meetings took place to start developing the specification.
+
+## Specification development: join the conversation!
+
+The Open Podcast API initiative builds on open source and open standards. Contributors from different projects work in the open to develop the specifications. All projects and products, from open source to proprietary, are encouraged to join us in developing the specs. Only then we can ensure we work towards the aim of a set of interoperable specs.
+
+[Join the chatroom on Matrix](https://matrix.to/#/!ZHdcrdWSgxXRREuJdU:matrix.org) | [Give input in our GitHub discussions](https://github.com/orgs/OpenPodcastAPI/discussions/categories/problem-definitions-solutions) | [Add our monthly meeting to your calendar](https://github.com/orgs/OpenPodcastAPI/discussions/15)
+
+[^gPoddermaintainer] [Call on the gPodder.net repository for a new maintainer](https://github.com/gpodder/mygpo/blob/81e3d13e00e6c3d6db7ee6a22734041bf6fde128/maintainer-needed.md)
+[^APforumGpodderIssues] [Questions about gPodder.net issues on the AntennaPod forum](https://forum.antennapod.org/t/problem-with-gpodder-net/374)
+[^gPodder50x] [Issue in the gPodder.net repository on 502 errors](https://github.com/gpodder/mygpo/issues/527)
+[^APforumGpodderRemoval] [Suggestion on the AntennaPod forum to remove gPodder.net support](https://forum.antennapod.org/t/should-gpodder-net-be-supported-long-term/396)
+[^initialAPIdiscussions] [Initial discussions on the needs for a new podcast synchronisation API](https://github.com/thrillfall/nextcloud-gpodder/discussions/91)
```

Comment:
```suggestion
[^gPoddermaintainer]: [Call on the gPodder.net repository for a new maintainer](https://github.com/gpodder/mygpo/blob/81e3d13e00e6c3d6db7ee6a22734041bf6fde128/maintainer-needed.md)
[^APforumGpodderIssues]: [Questions about gPodder.net issues on the AntennaPod forum](https://forum.antennapod.org/t/problem-with-gpodder-net/374)
[^gPodder50x]: [Issue in the gPodder.net repository on 502 errors](https://github.com/gpodder/mygpo/issues/527)
[^APforumGpodderRemoval]: [Suggestion on the AntennaPod forum to remove gPodder.net support](https://forum.antennapod.org/t/should-gpodder-net-be-supported-long-term/396)
[^initialAPIdiscussions]: [Initial discussions on the needs for a new podcast synchronisation API](https://github.com/thrillfall/nextcloud-gpodder/discussions/91)
```

Corrected the syntax, taken from https://myst-parser.readthedocs.io/en/latest/syntax/typography.html#footnotes
Otherwise, it wouldn't be parsed as footnotes.